packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -407,6 +408,23 @@ class SemanticsProperties extends DiagnosticableTree {
   /// Doing so instructs screen readers to not read out the [value].
   final bool obscured;
 
+  /// If non-null, whether the node corresponds to a visually distinct subtree.
+  ///
+  /// When the most specific route semantics node changes, the platform
+  /// accessibility framework will make an edge-triggered announcement and clear
+  /// the current accessibility focus. Specificity is defined as the last child
+  /// in a pre-order traversal - left to right, root first.
```

Comment:
not sure what "left to right" means here, can you elaborate?

Reply:
Updated, right now it is the last child in inverse hit-test order